css/ampstart-base/base-page.css

```diff
@@ -24,13 +24,19 @@ body {
   background: var(--color-body-background);
   color: var(--color-font-default);
   line-height: var(--line-height-3);
-  font-family: var(--font-family);
+  font-family: var(--base-font-family);
   min-width: var(--body-min-width);
   overflow-x: hidden;
   font-smooth: always;
   -webkit-font-smoothing: antialiased;
 }
 
+/* This is needed to override from templates*/
```

Comment:
I was trying to have the fallback font be what we declared (Per UX). 
The build process kills this if there is no override and uses the body selector. Hence this works.